### PR TITLE
Prepare the redirection of this website

### PR DESCRIPTION
**Pull request by elopio**

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -1,3 +1,15 @@
 [build]
 command = "npm run docsite build"
 publish = "docsite-build"
+
+[[redirects]]
+  from = "/"
```
Comment (elopio) [inline netlify.toml]: @frangio I think it should be "/*"

File: netlify.toml [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+  to = "https://docs.openzeppelin.com/"
```
Comment (elopio) [inline netlify.toml]: Why the home page, and not /contracts?

File: netlify.toml (hunk 1 above)
Comment (frangio) [reply netlify.toml]: I'd prefer to keep this conservative redirect for now.

File: netlify.toml (hunk 2 above)
Comment (frangio) [reply netlify.toml]: Fixed.

**Pull request merged**